README.md

```diff
@@ -87,13 +62,20 @@ The key features of this project are:
 - Use of flexbox to replicate a template
 - An adaptive layout that responds to screen width
 
+### Features implemented
+
+- Functioning mobile version with a responsive layout
+- Desktop version (min 768px) working complete with animations
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Coming soon...
+To see my Portfolio in action, simply head to
+
+> tomasesquivelgc.github.io/index
```

Comment:
[Optional] I would like to suggest you insert a link text here instead of the full URL.👍
```suggestion
To see my Portfolio in action, simply check [here](tomasesquivelgc.github.io/index)
```